server/routes/overview/prepareTimelineForView.test.ts

```diff
@@ -0,0 +1,199 @@
+import { parseISO } from 'date-fns'
+import type { Timeline, TimelineEvent } from 'viewModels'
+import prepareTimelineForView from './prepareTimelineForView'
+
+describe('prepareTimelineForView', () => {
```

Comment:
This looks like a new file but is actually a rename and move of `timelineResolver.test.ts`